lib/screens/v2/candidate_generator/widgets/reconstructed_alert.ex

```diff
@@ -74,4 +78,19 @@ defmodule Screens.V2.CandidateGenerator.Widgets.ReconstructedAlert do
           false
       end
   end
+
+  defp get_stations(alert, fetch_stop_name_fn) do
+    %{alert: alert}
+    |> BaseAlert.informed_entities()
+    |> Enum.map(fn %{stop: stop_id} -> stop_id end)
+    |> Enum.filter(&String.starts_with?(&1, "place-"))
+    |> Enum.uniq()
+    |> Enum.map(
+      &case fetch_stop_name_fn.(&1) do
+        :error -> []
+        name -> name
+      end
+    )
```

Comment:
Oop, in order to have stops whose names we failed to fetch disappear completely from the list this would need to be
```suggestion
    |> Enum.flat_map(
      &case fetch_stop_name_fn.(&1) do
        :error -> []
        name -> [name]
      end
    )
```

Some examples [here in the docs](https://hexdocs.pm/elixir/Enum.html#flat_map/2).

Reply:
Ope, I applied your change, and then merged from master and lost the flat_map. I didn't write line 91 correctly though. Thanks for catching both of these!